Programs/Game/Rock Paper Scissors With Twist.c

```diff
@@ -156,85 +99,57 @@ void playerKeyEvent()
             {
                 if(moves == 3)
                 {
-                    if(key == 'q')
-                        playerOneMoves = 1;
-                    else if(key == 'w')
-                        playerOneMoves = 2;
-                    else if(key == 'e')
-                        playerOneMoves = 3;
+           	    for (int i =0;i<3;i++)
+		    {
+		       if(key == keys1[i])
+                          playerTwoMoves = i+1;
```

Comment:
where are playerOneMoves store since now the while loop forever since playerOneMoves are never stored

Reply:
playerOneMoves is a global variable at line 41 I didn't alter any global variables from the original.